e2e/fixtures/simple-send/state.json

```diff
@@ -0,0 +1 @@
(1) year after such claim or cause of action arose or be forever barred.\n\n### 14.4 Section Titles ###\n\nThe section titles in the Terms are for convenience only and have no legal or contractual effect.\n\n### 14.5 Communications ###\n\nUsers with questions, complaints or claims with respect to the Service may contact us using the relevant contact information set forth above and at communications@metamask.io.\n\n## 15 Related Links ##\n\n**[Terms of Use](https://metamask.io/terms.html)**\n\n**[Privacy](https://metamask.io/privacy.html)**\n\n**[Attributions](https://metamask.io/attributions.html)**\n","date":"Thu Feb 09 2017","id":0,"read":false,"title":"Terms of Use"},{"body":"MetaMask is beta software. \n\nWhen you log in to MetaMask and approve account access, your current account's address is visible to the site you're currently viewing. This can be used to look up your account balances of Ether and other tokens.\n\nFor your privacy, take caution when approving account access and sign out of MetaMask when you're done using a site.\n\n","date":"Mon May 08 2017","id":2,"read":false,"title":"Privacy Notice"},{"body":"Dear MetaMask Users,\n\nThere have been several instances of high-profile legitimate websites such as BTC Manager and Games Workshop that have had their websites temporarily compromised. This involves showing a fake MetaMask window on the page asking for user's seed phrases. MetaMask will never open itself to ask you for your seed phrase, and users are encouraged to report these instances immediately to either [our phishing blacklist](https://github.com/MetaMask/eth-phishing-detect/issues) or our support email at [support@metamask.io](mailto:support@metamask.io).\n\n[Please read our full article on this ongoing issue over at Medium.](https://medium.com/metamask/new-phishing-strategy-becoming-common-1b1123837168)\n","date":"Wed Jun 13 2018","id":4,"read":false,"title":"Phishing Warning"}]},"PreferencesController":{"accountTokens":{"0x7ecf4659ab223cf48a9b082eba8041c5f4b1d45e":{"localhost":[]},"0x985c30949c92df7a0bd42e0f3e3d539ece98db24":{"localhost":[],"mainnet":[]}},"assetImages":{},"completedOnboarding":true,"completedUiMigration":true,"currentAccountTab":"history","currentLocale":"en","featureFlags":{},"firstTimeFlowType":"import","forgottenPassword":false,"frequentRpcListDetail":[],"identities":{"0x7ecf4659ab223cf48a9b082eba8041c5f4b1d45e":{"address":"0x7ecf4659ab223cf48a9b082eba8041c5f4b1d45e","name":"Account 2"},"0x985c30949c92df7a0bd42e0f3e3d539ece98db24":{"address":"0x985c30949c92df7a0bd42e0f3e3d539ece98db24","name":"Account 1"}},"knownMethodData":{},"lostIdentities":{},"metaMetricsId":null,"metaMetricsSendCount":0,"participateInMetaMetrics":false,"preferences":{"useNativeCurrencyAsPrimaryCurrency":true},"seedWords":null,"selectedAddress":"0x7ecf4659ab223cf48a9b082eba8041c5f4b1d45e","suggestedTokens":{},"tokens":[],"useBlockie":false},"config":{},"firstTimeInfo":{"date":1559146971733,"version":"6.2.2"}},"meta":{"version":31}}
```

Comment:
We should probably pretty-print these as a rule. It'd be easier to read and would reduce diff noise when they need to be updated.

Reply:
On that note, we should probably have a script to help with generating these state fixtures - something we could call from the browser console perhaps. We could probably write a simple function and stick it on `window` if `METAMASK_DEBUG` is set - something like that anyway. Not something that needs to be in this PR though of course.